content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
-==================================
-Configure Amazon Connector in Odoo
-==================================
+==============================
+Amazon Connector configuration
+==============================
 
-Register your Amazon account in Odoo
-====================================
+Odoo allows users to register an Amazon seller account in the database, but the user **must** have
+a paid Amazon Seller account prior to completing the configuration.
+
+To learn how to set up a paid Seller account on Amazon, log in to Amazon, and select *Start a
+Selling Account* from the *Account & Lists* drop-down menu, located in the header. Once that process
+is complete, proceed to follow the instructions below to register and link that Amazon Seller
+account in Odoo.
```

Comment:
Is there any Amazon documentation that we can link to here in this sentence? Or will they just have to log into Amazon and see for themselves?

Reply:
Yeah, unfortunately, they would have to log into Amazon and go from there. But I fully understand where you're coming from.